Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -623,7 +624,7 @@
 				MACOSX_DEPLOYMENT_TARGET = 13.4;
 				MARKETING_VERSION = 1.0;
 				OTHER_LDFLAGS = "-ObjC";
-				PRODUCT_BUNDLE_IDENTIFIER = "com.icyappstudio.Basic-Car-Maintenance";
+				PRODUCT_BUNDLE_IDENTIFIER = "ch.appfros.Basic-Car-Maintenance";
```

Comment:
Don't change the bundle ID please! Go back and fix all the instances of this

Reply:
Sorry, that's quite obviously not a good idea... As per our small conversation on Mastadon, I changed it cause it gave me an error for macOS–but committing it in the PR doesn't make sense anyway…